src/checks/lowercase_key.rs

```diff
@@ -0,0 +1,51 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct LowercaseKeyChecker {}
+
+impl Check for LowercaseKeyChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        let line_str: Vec<&str> = line.raw_string.split('=').collect();
+        let key = line_str[0];
+        if key.to_uppercase() == key {
+            None
+        } else {
+            Some(Warning {
+                message: format!("The {} key should be in uppercase", key),
```

Comment:
Please, use the field `warning` to store and retrieve a message:
```suggestion
                message: format!(self.warning.clone(), key),
```

Reply:
https://github.com/mgrachev/dotenv-linter/pull/21#discussion_r358616840